### PR TITLE
fix: resolved dataset upload

### PR DESCRIPTION
**Pull request by manomage**
•⁠  ⁠Updated the path to DataDetails page to use normal routing convention - /data/:id
•⁠  ⁠Moved the timeseries slice into data slice.
•⁠  ⁠Created a util to convert both pre-processed timeseries and views to chart data.
•⁠  ⁠Resolved display chart for uploading dataset.
•⁠  ⁠Setup a logic to map out views data based on charts created when uploading dataset

### CODE REVIEW

File: src/pages/data/components/BrowseDataTable.tsx
Comment (ricardocr987) [inline src/pages/data/components/BrowseDataTable.tsx:1]: merge changes on your branch from main to get this right

Comment (manomage) [reply src/pages/data/components/BrowseDataTable.tsx:1]: I merged already but changed the route for DataDetails page. Refer to this file `src/routes.tsx` to see the updated path

**Comment by MHHukiewitz** [comment]
In future, please provide a short description of your changes and rationale behind them, besides in the title of the PR.

**Comment by manomage** [comment]
> @manomage In future, please provide a short description of your changes and rationale behind them, besides in the title of the PR.

Noted.  I'll be adding a description to this PR shortly

**Review by MHHukiewitz — APPROVED** [review]

**Pull request merged**